cmake/features.cmake

```diff
@@ -68,3 +68,5 @@ Usage: -DSELECTIVE_BUILD=ON -DSELECTIVE_BUILD_STAT=/path/*.csv" OFF
                ALLOWED_VALUES ON OFF COLLECT)
 
 set(LINKCHECKER_PY "" CACHE FILEPATH "Path to linkchecker.py for documentation check")
+
+ie_option(ENABLE_ERROR_HIGHLIGHT "Highlight errors and warnings during compile time" OFF)
```

Comment:
why do we need an option? It can be enabled for all cases I believe.

Reply:
For errors highlighting I use ANSI escape codes. In my opinion, it looks a little ugly in the CI or IDE logs. It useful only in terminals where ANSI escape codes are supported.